### PR TITLE
4.7.27 releasenotes

### PR DESCRIPTION
**Pull request by agh1**
I'm putting the raw output of the script up early to help people navigate the release.  @alifrumin and I will be working on this progressively through the end of the month.

Please note that the headers are automatically generated from JIRA categories.  They are wrong 50% of the time.

### CODE REVIEW

**Comment by totten** [comment]
@agh1, the `4.7.26` release has been renumbered to `4.7.27`. I've updated the base-branch of this PR, but it the content should updated to reference `4.7.27`.

File: release-notes/4.7.26.md [hunk 1]
```diff
@@ -0,0 +1,361 @@
+# CiviCRM 4.7.27
```
Comment (seamuslee001) [inline release-notes/4.7.26.md]: @agh1 file name looks wrong

Comment (agh1) [reply release-notes/4.7.26.md]: Thanks @seamuslee001 I had some file name mayhem last night.  Fixed it now.

File: release-notes/4.7.27.md [hunk 2]
```diff
@@ -0,0 +1,436 @@
...
+
+### Core CiviCRM
+
+- **[CRM-21229](https://issues.civicrm.org/jira/browse/CRM-21229) Manage Group
+  page is slow if you have smart groups
+  ([11033](https://github.com/civicrm/civicrm-core/pull/11033)) (partially
+  resolved)**
+
+  The Manage Groups page now will display "Unknown" for the count of smart group
+  members where the smart group cache is empty.  You can click the button to
```
Comment (monishdeb) [inline release-notes/4.7.27.md]: @agh1 actually the "Unkown" is set for the count when the smart group cache got outdated. Can you change ```empty``` to ```outdated``` here?

Comment (agh1) [reply release-notes/4.7.27.md]: Absolutely--thanks for catching this distinction!

**Comment by agh1** [comment]
So I need some way to work this into the release notes:
screenshot-2017-10-31 instagram post by laura medalia oct 27 2017 at 3 52pm utc
https://www.instagram.com/p/BawdVh8F0x-/

**Comment by agh1** [comment]
@totten this is all ready

**Pull request merged**